cmd/registry/conformance/linter_test.go

```diff
@@ -0,0 +1,292 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package conformance
+
+import (
+	"testing"
+
+	"github.com/apigee/registry/rpc"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"google.golang.org/protobuf/testing/protocmp"
+)
+
+var rules = []*rpc.Rule{
```

Comment:
Can we declare these separately in a var block with descriptive names for each rule? As long as they're only being used as "drop-in" values (whose contents don't actually matter) I think it's fine that they're not in-lined in the test. Still, it would be helpful to have a general idea what `rule[0]` actually is, e.g. `noRefSiblingsRule`.

Reply:
Done.